Library/Homebrew/rubocops/urls.rb

```diff
@@ -22,6 +22,7 @@ class Urls < FormulaCop
           https://osxbook.com/book/bonus/chapter8/core/download/gcore
           https://naif.jpl.nasa.gov/pub/naif/toolkit/C/MacIntel_OSX_AppleC_64bit/packages/
           https://artifacts.videolan.org/x264/release-macos/
+          https://github.com/vifm/vifm/releases/download
```

Comment:
```suggestion
          https://github.com/vifm/vifm/releases/download/v0.11/vifm-osx-0.11.tar.bz2
```

as this seems specific to a single release.

Reply:
yeah, that is right, only for this release.